utils/scripts/install-el8.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -e
+#!/usr/bin/env bash
```

Comment:
Seems an odd change, is arm really that weird?

Reply:
Yes, check patch is a PITA with sh. I am finally going to move everything to bash to remove all the non-sense checkpatch warning.

Reply:
I don't know why shellcheck is being such a pain with this script.  It clearly should work in sh and it seems a shame to switch to bash just to keep it happy.